### DISCUSSION TITLE
rails-ui-kits

### DISCUSSION THREAD

**Discussion by giscus** [yshmarov/yshmarov.github.io#7] [General]
# rails-ui-kits

Building backends with Rails is amazingly fast.

https://blog.corsego.com/rails-ui-kits

**Comment by yshmarov** [comment]
finally! giscos comments!

**Comment by secretpray** [comment]
Це посилання 
http://rapidrails.cc 
не робоче!
Модно додати ще й Flowbite

**Reply by yshmarov** [reply]
works for me... looks like it might have some geo-restrictions
<img width="1451" alt="Screenshot 2024-11-16 at 23 19 16" src="https://github.com/user-attachments/assets/88ec213a-436c-47a6-b7f7-1ba4af8c2dae">

**Reply by AhmedNadar** [reply]
Github comment is much better than old system 😀
The website is working fine both the components library https://rapidrails.cc/components and the agency https://rapidrails.cc

**Reply by secretpray** [reply]
/Users/secretpray/Desktop/Screen Recording 2024-11-17 at 18.17.54.mov

**Reply by yshmarov** [reply]
yep my mistake. I typed in the link wrong. fixed now :)

**Comment by secretpray** [comment]
Meraki UI 
https://merakiui.com/ 

daisyUI 
https://daisyui.com/

**Reply by yshmarov** [reply]
Here I list only Rails-specific libraries, not just any UI component libs
